extensions/wikia/ArticleMetaDescription/ArticleMetaDescription.php

```diff
@@ -35,7 +35,7 @@
  * @param string $text
  * @return bool
  */
-function wfArticleMetaDescription( &$out, &$text ) {
+function wfArticleMetaDescription( OutputPage $out, &$text ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$text". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)